LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -4194,6 +4194,48 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
                             c.Add(new Control("Fan #" + (i + 1), i));
 
                         break;
+                    
+                    case Model.PROART_B760_CREATOR_D4: // NCT6798D
+                        v.Add(new Voltage("Vcore", 0));
+                        v.Add(new Voltage("+5V", 1, 4, 1));
+                        v.Add(new Voltage("AVSB", 2, 34, 34));
+                        v.Add(new Voltage("3VCC", 3, 34, 34));
+                        v.Add(new Voltage("+12V", 4, 11, 1));
+                        v.Add(new Voltage("IVR Atom L2 Cluster #1", 5));
+                        v.Add(new Voltage("Voltage #7", 6));
+                        v.Add(new Voltage("+3V Standby", 7, 34, 34));
+                        v.Add(new Voltage("CMOS Battery", 8, 34, 34));
+                        v.Add(new Voltage("CPU Termination", 9, 1, 1));
+                        v.Add(new Voltage("Voltage #11", 10));
+                        v.Add(new Voltage("IVR Atom L2 Cluster #0", 11, 1, 1));
+                        v.Add(new Voltage("PCH", 12));
+                        v.Add(new Voltage("Voltage #14", 13));
+                        v.Add(new Voltage("Voltage #15", 14));
+                        
+                        t.Add(new Temperature("CPU Package", 0)); // PECI_0, CPU Package
+                        t.Add(new Temperature("CPU", 1)); // CPUTIN, CPU
+                        t.Add(new Temperature("Motherboard", 2)); // SYSTIN, MOTHERBOARD
+                        t.Add(new Temperature("T Sensor", 8)); // TSENSOR
+                        t.Add(new Temperature("PCH", 13)); // PCH_CHIP_TEMP
+                        t.Add(new Temperature("PECI_0_CAL", 22)); // PECI_0_CAL, CPU
```

Comment:
Is there a better name for this?

Reply:
1. I'm not quite sure about PECI_0_CAL (suppose, CAL means "Calculated" or even "Average"), but it definitely some CPU temp.

2. "T Sensor"/"TSENSOR" - well, according to motherboard specs, it is...temperature sensor, called by Asus as "Thermal Sensor Header".
Here is the link for their [support site](https://www.asus.com/motherboards-components/motherboards/proart/proart-b760-creator-d4/helpdesk_manual?model2Name=ProArt-B760-CREATOR-D4).
Direct link to motherboard [user's manual](https://dlcdnets.asus.com/pub/ASUS/mb/LGA1700/ProArt_B760-CREATOR_D4/E21433_ProArt_B760-CREATOR_D4_UM_WEB.pdf?model=ProArt%20B760-CREATOR%20D4)
_*Not trying to waste your time, just in case you want to see their naming._

3. "Motherboard"/"SYSTIN" seems to be typical motherboard sensor, but i'm not sure where it exactly placed.
Will try to find out and reply.

4. "PCH"/"PCH_CHIP_TEMP" - chipset temperature, i really don't know how to name it more accurate/correct.
Seems to me it's pretty self-explanatory.

--

If you can point me out to some naming guideline or "see this mobo, use it naming" - no problem.
I suppose "T Sensor" probably should be named as "Thermal Sensor Header", but it's Asus naming (or, let's say - extended naming), not common (afaik).

Or, if you can say "debug there, show output" or something like that - again, no problem.
I own this motherboard right now and can debug needed stuff.

--

Or do you mean voltages, like "Voltage #X"? 
I took them from another motherboard on same chip (NCT6798D), slightly checked to match other tools readings, but probably can try to match with BIOS/other tools, if it needed.